src/main/java/eu/dissco/dataexporter/controller/DataExporterController.java

```diff
@@ -1,8 +1,56 @@
 package eu.dissco.dataexporter.controller;
 
+import eu.dissco.dataexporter.exception.ForbiddenException;
+import eu.dissco.dataexporter.exception.InvalidRequestException;
+import eu.dissco.dataexporter.schema.ExportJobRequest;
+import eu.dissco.dataexporter.service.DataExporterService;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.jwt.Jwt;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
+@RequestMapping("")
+@Slf4j
+@RequiredArgsConstructor
 public class DataExporterController {
 
+  private final DataExporterService service;
+
+  @PostMapping("/schedule")
+  public ResponseEntity<Void> scheduleJob(Authentication authentication,
+      @RequestBody ExportJobRequest request)
+      throws ForbiddenException, InvalidRequestException {
+    var orcid = getOrcid(authentication);
+    service.addJobToQueue(request, orcid);
+    log.info("Successfully posted job to queue");
+    return ResponseEntity.status(HttpStatus.ACCEPTED).build();
+  }
+
+  @PostMapping("/{id}/running")
+  public ResponseEntity<Void> markJobAsRunning(@PathVariable("id") UUID id) {
```

Comment:
Ok yeah and we won't expose this endpoint, otherwise anyone with an id can put the status un running :smiling_face_with_tear:  But if we keep this endpoint internal (so not ingressRoute to it) then we should be ok. Otherwise we could make a role for it and assign it to the export jobs, probably better in the future

Reply:
Should i make a separate controller for the internal endpoints? or give them a prefix like `/internal/{id}/running` so if we want to have more internal endpoints we can just match that pattern

Reply:
hmm maybe yeah for now that might be good. Otherwise we should do the role thing